cmd/cmd_synctarget.go

```diff
@@ -0,0 +1,156 @@
+package cmd
+
+import (
+	"context"
+	"github.com/lithictech/webhookdb-cli/appcontext"
+	"github.com/lithictech/webhookdb-cli/client"
+	"github.com/urfave/cli/v2"
+)
+
+var synctargetCmd = &cli.Command{
+	Name:  "sync-target",
+	Usage: "",
+	Subcommands: []*cli.Command{
+		{
+			Name:  "create",
+			Usage: "Create a sync target for the given organization.",
+			Flags: []cli.Flag{
+				orgFlag(),
+				integrationFlag(),
+				connectionUrlFlag(),
+				periodFlag(),
+				schemaFlag(),
+				tableFlag(),
+			},
+			Action: cliAction(func(c *cli.Context, ac appcontext.AppContext, ctx context.Context) error {
+				input := client.SyncTargetCreateInput{
+					OrgIdentifier:       getOrgFlag(c, ac.Prefs),
+					IntegrationOpaqueId: c.String("integration"),
+					ConnectionUrl:       c.String("connection-url"),
+					Period:              c.Int("period"),
+					Schema:              c.String("schema"),
+					Table:               c.String("table"),
+				}
+				out, err := client.SyncTargetCreate(ctx, ac.Auth, input)
+				if err != nil {
+					return err
+				}
+				printlnif(c, out.Message(), false)
+				return nil
+			}),
+		},
+		{
+			Name:  "delete",
+			Usage: "Delete a sync target.",
```

Comment:
Delete the sync target and stop any further syncing. The table being synced to is not modified.